internal/server/web/proxy/run.go

```diff
@@ -75,7 +73,6 @@ func logRunResponse(log *zap.Logger, data []byte, prod, private bool, cid string
 func logRetrieveRunRequest(log *zap.Logger, prod bool, cid, tid, rid string) {
```

Comment:
`cid` can also be removed from the function header for all the functions inside this file